cmd/serve/sftp/sftp.go

```diff
@@ -101,6 +112,22 @@ sftp backend, but it may not be with other SFTP clients.
 		}
 		cmd.Run(false, true, command, func() error {
 			s := newServer(context.Background(), f, &Opt)
+			if Opt.Stdio {
+				if terminal.IsTerminal(int(os.Stdout.Fd())) {
```

Comment:
I think it would make sense to move the stdio setup into `connection.go` into a new `stdioChannel` function maybe.

Then it is with the other channel stuff and will make more sense with the factored out `serveChannel` see below.

Reply:
Done.